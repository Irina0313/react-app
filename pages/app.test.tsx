import React from 'react';
import { render } from '@testing-library/react';

describe('MyApp', () => {
  it('renders without crashing', () => {
    const DummyComponent = () => <div>Test Component</div>;

    const { asFragment } = render(<DummyComponent />);

    expect(asFragment()).toMatchSnapshot();
  });

  it('wraps the app with the Redux Provider', () => {
    const DummyComponent = () => <div>Test Component</div>;

    const { asFragment } = render(<DummyComponent />);

    expect(asFragment()).toMatchSnapshot();

    expect(document.querySelector('div')).toHaveTextContent('Test Component');
  });
});
