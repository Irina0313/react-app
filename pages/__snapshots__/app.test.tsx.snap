// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyApp renders without crashing 1`] = `
<DocumentFragment>
  <div>
    Test Component
  </div>
</DocumentFragment>
`;

exports[`MyApp wraps the app with the Redux Provider 1`] = `
<DocumentFragment>
  <div>
    Test Component
  </div>
</DocumentFragment>
`;
